### ISSUE TITLE
The Windows hosts file CANNOT block I.P. addresses.

### ISSUE THREAD

**Issue by Lister0fSmeg** [Ruddernation-Designs/Adobe-URL-Block-List#24]
The Windows hosts file CANNOT block I.P. addresses. It only works for Domains. If you find the I.P. address of a website and then put that in the hosts files with either 127.0.0.1 or 0.0.0.0 you will still be able to access the website with your browser.

There is no point putting I.P. addresses in the windows hosts file because they will not work.

**Comment by polypores** [comment]
Solution 1: Restart Windows or disable & reenable your network interface in Control Panel.
Solution 2. Clear Windows caches name resolution data:
`ipconfig /flushdns`

**Comment by valipod** [comment]
No, this is not something that ListerOfSmeg needs help with, he signals that it is completely useless to add IP addresses to the hosts. It's basic understanding of DNS and what the hosts file does.
